### ISSUE TITLE
[Accessibility] Please add aria alert for "No online Collaborators" message

### ISSUE THREAD

**Issue by jooyoungseo** [iamhyc/Overleaf-Workshop#48]
Type: <b>Feature Request</b>

When there is no collaborator in the current Overleaf session and users execute `Overleaf Workshop: Jump to Collaborator ...` command, they will see the following message:

> No online Collaborators

However, this message is not auto-announced to screen reader because it does not have alert attribute. I think the easiest solution would be adding `aria-live="assertive"` to the message div.

Extension version: 0.5.3
VS Code version: Code - Insiders 1.85.0-insider (94a3d7bac770b43ce97e9685715bef1a60d1a759, 2023-12-01T19:00:37.339Z)
OS version: Windows_NT x64 10.0.22631
Modes:

**Comment by iamhyc** [comment]
> However, this message is not auto-announced to screen reader because it does not have alert attribute.
Got it. I thought vscode had well coped with the popup windows for accessbility. Will remove it in next release.

> I think the easiest solution would be adding aria-live="assertive" to the message div.
The popup window is generated by vscode and I cannot add extra property. Any better suggestions?

**Comment by jooyoungseo** [comment]
@iamhyc -- This works so nicely, and auto anounces when there is no collaborator. Thanks so much!
